contracts/src/interfaces/IEverlongPositions.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+interface IEverlongPositions {
+    /// @dev Tracks the total amount of bonds managed by Everlong
+    ///      with the same maturityTime.
+    ///      TODO: Update this comment if multiple positions are needed to obtain
+    ///            the total amount of bonds managed by Everlong for a maturity time.
```

Comment:
This won't be needed. Under the hood, Hyperdrive positions are grouped together by maturity time. There is no reason why we shouldn't do that here. It's cheaper from a storage perspective, and it reduces DOS vectors.